### ISSUE TITLE
Sidebar not opening when highlighted text is clicked (in Firefox, for one website)

### ISSUE THREAD

**Issue by klemay** [hypothesis/client#723]
### Steps to reproduce
1. In Firefox (for Mac or PC), visit [this web page](https://wisc.pb.unizin.org/frenchcscr/chapter/assignment-alain-mabanckou-black-bazar/) - or any article on the site with annotations.
2. Click on some highlighted text.

### Expected behaviour
The Hypothesis sidebar should open, focused on the annotation associated with the highlighted text.

### Actual behaviour
Most of the time, nothing. On some page loads, the sidebar will open as expected, but most of the time it will not. 

### Browser/system information
Firefox 59.0.2 on the latest version of Mac OS (though user who reported this issue says it happens for PC users as well)

### Additional details
- This came out of a Zendesk ticket: https://hypothesis.zendesk.com/agent/tickets/2429
- The site is running Pressbooks (a WordPress plugin) and uses the Hypothesis WordPress plugin to add the H client to pages by default
- @robertknight noted that he was able to reproduce in Firefox stable and nightly that the issue was not consistent. On some page loads clicking highlights did open the sidebar on others it did not. 
- @judell  noted that this page uses a custom build of Hypothesis, but I don’t think that is related. None of the code that is different in the “H5P” version of Hypothesis is related to selecting annotations.

**Comment by SteelWagstaff** [comment]
Any updates on what might be happening here?

**Comment by gavento** [comment]
I was testing this issue on Mozilla Firefox 64.0 (PC Core i5, Ubuntu 18.04) and could not reproduce it even once.
Tried the following ~20 times in total: reloading while not logged in, reloading in private window, reloading when logged in (in all cases at least once while clearing caches).
The sidebar handle always appeared after reload and clicking a highlighted text always correctly opened the annotation in the sidebar.

**Comment by klemay** [comment]
@gavento I just tried it again with Firefox 64.0 for Mac, and had the same results - clicking on highlighted text now results in the sidebar opening as expected, even after trying it several times under the circumstances you listed (logged out, logged in, private window, cache cleared)

I wonder if this was fixed as part of recent work we've done on the bug backlog, or with one of Firefox's recent updates. 

@robertknight or @judell are you aware of any recent changes to our client (or Firefox) that could explain this? Do we think it's safe to close this issue?
